variables.tf

```diff
@@ -0,0 +1,15 @@
+variable "prefix" {
+  description = "The prefix used for all resources in this environment"
+}
+
+variable "location" {
+  description = "The Azure location where all resources in this deployment should be created"
+  default     = "uksouth"
+}
+
+variable "app-authentication-disabled" { default = true }
+variable "app-flask-secret-key" {}
```

Comment:
These aren't being leaked currently, but variables can also have `sensitive = true` set to ensure they don't get logged.